src/components/forms/Combobox.tsx

```diff
@@ -286,12 +294,8 @@ export function ComboboxStatic({
                   }}
                   key={entry.id}
                   value={entry}
-                  className={() =>
-                    classNames(
-                      'min-w-[19rem] relative cursor-default select-none py-2 pl-3 pr-9'
-                      // active ? 'bg-gray-100 text-gray-900' : 'text-gray-900'
-                    )
-                  }
+                  className="min-w-[19rem] relative cursor-default select-none py-2 pl-3 pr-9"
+                  // active ? 'bg-gray-100 text-gray-900' : 'text-gray-900'
```

Comment:
@beganovich We had a weird result in the class list due to the callback function. Screenshot:

<img width="657" alt="Screenshot 2023-09-08 at 10 13 06" src="https://github.com/invoiceninja/ui/assets/51542191/e2d077c9-ddf0-4471-a96f-e80a22a6e805">

We actually do not need the `classNames` function in this case, so I just removed it to avoid this behavior. Let me know your thoughts.